cypress/package.json

```diff
@@ -15,17 +15,17 @@
   },
   "devDependencies": {
```

Comment:
would be nice if we can automate checking for this. For example, on each build, we get a log with all duplicated dependencies.